### PR TITLE
feat(o-shop):added login-form

### PR DESCRIPTION
**Pull request by m-spilman**

### CODE REVIEW

File: src/components/Auth/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    username: '',
+    password: '',
+  });
+
+  const [submitted, setSubmitted] = useState(false);
+  const { username, password } = inputs;
+
+  const onSubmit = (e) => {
+    setSubmitted(true);
+    console.log('submit');
```
Comment (ahaly92) [inline src/components/Auth/LoginForm.js]: remove console logs for PRs

Comment (BarretteBanner) [inline src/components/Auth/LoginForm.js]: Remove console.logs

File: src/components/Auth/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+function LoginForm() {
+  const [inputs, setInputs] = useState({
+    username: '',
+    password: '',
+  });
+
+  const [submitted, setSubmitted] = useState(false);
+  const { username, password } = inputs;
+
+  const onSubmit = (e) => {
```
Comment (ahaly92) [inline src/components/Auth/LoginForm.js]: do not abbreviate. Lets spell out e -> event for readability

File: src/components/Auth/LoginForm.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            value={username}
+            onChange={onChange}
+            className={
+              'form-control' + (submitted && !username ? ' is-valid' : '')
+            }
+          />
+          {submitted && !username && (
+            <div className="invalid-feedback d-block">Username is required</div>
+          )}
+        </div>{' '}
```
Comment (ahaly92) [inline src/components/Auth/LoginForm.js]: do you need {' '}

File: src/components/Auth/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            type="text"
+            name="username"
+            placeholder="Username"
+            value={username}
+            onChange={onChange}
+            className={
+              'form-control' + (submitted && !username ? ' is-valid' : '')
+            }
+          />
+          {submitted && !username && (
```
Comment (BarretteBanner) [inline src/components/Auth/LoginForm.js:39]: do you need the second && ?

File: src/components/Auth/LoginForm.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            type="password"
+            name="password"
+            placeholder="Password"
+            value={password}
+            onChange={onChange}
+            className={
+              'form-control' + (submitted && !password ? ' invalid' : '')
+            }
+          />
+          {submitted && !password && (
```
Comment (BarretteBanner) [inline src/components/Auth/LoginForm.js]: same question here

File: src/pages/Login.js [hunk 6]
```diff
@@ -1,7 +1,10 @@
 import React from 'react';
+import LoginForm from '../components/Auth/LoginForm';
 
 function Login() {
-  return <h1>Login!</h1>;
+  //   return <h1>Login!</h1>;
```
Comment (BarretteBanner) [inline src/pages/Login.js]: remove comments

**Review by BarretteBanner — APPROVED** [review]

**Review by ahaly92 — APPROVED** [review]

**Review by ahaly92 — APPROVED** [review]

**Pull request merged**